pkg/runtime/runtime.go

```diff
@@ -39,7 +41,15 @@ func (it InterpreterType) String() string {
 type Interpreter struct {
 	Type    InterpreterType
 	Version *semver.Version
+
 	// The address of the main thread state for Python.
 	MainThreadAddress  uint64
 	InterpreterAddress uint64
 }
+
+type Type string
+
+type Runtime struct {
+	Type    Type
```

Comment:
nit: Maybe "name" is clearer

Reply:
I'm also planning to merge or embed this to the `Interpreter` type. I'll iterate on it in subsequent PRs.